config/ksql-server.properties

```diff
@@ -20,6 +20,7 @@
 # The default is any IPv4 interface on the machine.
 # NOTE: If set to wildcard or loopback set 'advertised.listener' to enable pull queries across machines
 listeners=http://0.0.0.0:8088
+ksql.query.pull.table.scan.enabled=true
```

Comment:
if we're comfortable putting this configuration in the main config properties, I think we should just default it to true. If there's a reason to doubt it, then I don't think we should put it as `true` here quite yet.

Reply:
Heh, that wasn't my intention.  This was a test configuration.